src/main/java/core/basesyntax/operation/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.operation;
+
+public interface OperationHandler {
+    void changeFruitQuantity(String fruit, int quantity);
```

Comment:
Change method name to more generic, like `handle`, because operation not necessary only changing the fruit quantity.